### PR TITLE
Updated help links

### PR DESCRIPTION
**Pull request by thakkaryash94**
replaced community links to discourse links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ Only a few users have access for for filling bugs in the tracker. New
 users should:
 
 1. Read ["How to ask for help or report an issue"](https://github.com/igniterealtime/Smack/wiki/How-to-ask-for-help,-report-an-issue-and-possible-solve-the-problem-yourself)
-2. [Create a forums account](https://community.igniterealtime.org/login!input.jspa?registerOnly=true) (only e-mail is a requirement, you can skip all the other fields).
+2. [Create a discourse account](https://discourse.igniterealtime.org/signup) (you can also sign up with your google account).
 3. Login to the forum account
```
Comment (vanitasvitae) [inline README.md]: This should probably say "Login to the account"

File: README.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ Only a few users have access for for filling bugs in the tracker. New
...
 4. Press New in your toolbar and choose Discussion
```
Comment (vanitasvitae) [inline README.md]: This step also changed

File: README.md (hunk 1 above)
Comment (thakkaryash94) [reply README.md]: bullet starts with "Create" so I suppose it means sign up not sign in!

Comment (vanitasvitae) [reply README.md]: I meant step 3 :)

Comment (thakkaryash94) [reply README.md]: I checked on discourse website, it is written as "discourse is a modern forum", So I keep it as it is. but to use our forum they need discourse account, that's why I wrote "Create a discourse account" and kept "Login to the forum account" as it is.

Comment (vanitasvitae) [reply README.md]: Ah okay 👍

**Pull request merged**